libMBIN/Source/NMS/GameComponents/CgExpeditionCategoryStrength.cs

```diff
@@ -5,8 +5,10 @@
 
 namespace libMBIN.NMS.GameComponents
 {
-    [NMS(Size = 0x14, GUID = 0x3B4F33D91D98F4AF, NameHash = 0xD22200F6DB5FF136, Broken = true)]
+    [NMS(Size = 0x14, GUID = 0x3B4F33D91D98F4AF, NameHash = 0xD22200F6DB5FF136)]
     public class CgExpeditionCategoryStrength : NMSTemplate
     {
+        [NMS(Size = 0x5)]
```

Comment:
does this have an associated enum value?

Reply:
corrected